README.md

```diff
@@ -84,11 +77,46 @@ Internally it uses [untyped](https://untyped.unjs.io/) and [jiti](https://github
 
     <!-- AUTOMD_END -->
 
-### Args supported for `jsdocs`
+#### Args supported for `jsdocs`
 
 - `src`: Path to the source file. The default is `./src/index` and can be omitted.
 - `headingLevel`: Nested level for markdown group headings (default is `2` => `##`). Note: Each function uses `headingLevel+1` for the title in nested levels.
-- `group`: Only render function exportes annotated with `@group name`. By default, there is no group filter. Value can be a string or array of strings.
+- `group`: Only render function exports annotated with `@group name`. By default, there is no group filter. Value can be a string or an array of strings.
+
+### `pm-install` Generator
+
+The `pm-install` generator generates commands for several JavaScript package managers.
+
+#### Usage
+
+    <!-- AUTOMD_START generator="pm-install" name="package-name" dev="true" -->
+    <!-- AUTOMD_END -->
+
+**Updated Result:**
+
+    <!-- AUTOMD_START generator="pm-install" name="package-name" dev="true" -->
+
+    ```sh
+    # npm
+    npm install -D package-name
+
+    # yarn
+    yarn add -D package-name
+
+    # pnpm
+    pnpm install -D package-name
+
+    # bun
+    bun install -D package-name
+    ```
+
+    <!-- AUTOMD_END -->
+
+#### Args supported for `pm-install`
+
+- `name`: The package name (by default tries to read from the `name` field in `package.json`).
```

Comment:
Thoughts on `packages` instead of `name`? I've often seen READMEs that suggest installing multiple at once (e.g. typescript-eslint suggests installing `typescript` as well).

```md
    <!-- AUTOMD_START generator="pm-install" packages="package-name, other-package-name" dev="true" -->

    ```sh
    # npm
    npm install -D package-name other-package-name

    # yarn
    yarn add -D package-name other-package-name

    # pnpm
    pnpm install -D package-name other-package-name

    # bun
    bun install -D package-name other-package-name
    ```

    <!-- AUTOMD_END -->
```

```md
    <!-- AUTOMD_START generator="pm-install" packages="package-name" dev="true" -->

    ```sh
    # npm
    npm install -D package-name

    # yarn
    yarn add -D package-name

    # pnpm
    pnpm install -D package-name

    # bun
    bun install -D package-name
    ```

    <!-- AUTOMD_END -->
```

Reply:
Hmm yes but it is also less common and I prefer to keep name shorter `pm` already suggests it is about package. We can easily also use `name="pkg1 pkg2"` btw even without array support wdyt?

Reply:
Well `pm` stands for "package manager" right? So it doesn't really suggest one way or another. How about we make it space separated though, you are right that comma-separated is a little much.

```html
    <!-- AUTOMD_START generator="pm-install" packages="pkg1 pkg2" -->
```

Reply:
i read `pm-install packages=` as `package manager install packages="..."` while `pm-install name=` as `package manger install name=`. 

More precisely what are setting in this param (according to npm) is "package specs"

Although we can really simply support `packages` as an alias if you strongly like it more feel free to make a new PR to propose (thinking to move this one forward faster)

Reply:
Sounds good yeah, I'll make a PR in the future!